industrial_ci/util.sh

```diff
@@ -35,13 +35,13 @@
 ## This is a script where the functions commonly used within the industrial_ci repo are defined.
 
 function travis_time_start {
-    set +x
+    if [ "$DEBUG_BASH" ] && [ "$DEBUG_BASH" == true ]; then set +x; fi
```

Comment:
Could this line be just `set +x` without if clause, because if `"$DEBUG_BASH" != true` then `set -x` must have not been done?